### ISSUE TITLE
Missing symbol

### ISSUE THREAD

**Issue by jmkeil** [HajoRijgersberg/OM#40]
Using a [comparison of unit ontologies with ABECTO](https://github.com/fusion-jena/abecto-unit-ontology-comparison) I became aware that the following unit has wrong symbol:
* [ ] http://www.ontology-of-units-of-measure.org/resource/om-2/astronomicalUnit: `au`

**Comment by HajoRijgersberg** [comment]
Thanks, but that's not entirely true: OM has the symbol 'AU'. 'au' is also used a lot in practice, so I should definitely define that symbol too, that's true. Now, it is unclear which symbol is the main symbol, so I don't know yet which one I will define as om:symbol and which one as om:alternativeSymbol. I'll dive further into this. First I'll simply add 'au' as additional om:alternativeSymbol, since 'AU' is already defined as om:symbol. But perhaps I have to swap them. I'll keep you updated!

**Comment by jmkeil** [comment]
* [UCUM](https://ucum.org/ucum.html#para-31) (which itself refers to ISO 1000, ISO 2955 and ANSI X3.50) states that `AU` is the correct value
*  **IAU: [Resolutions B1, B2, B3 and B4](https://www.iau.org/static/resolutions/IAU2012_English.pdf). Adopted at the General Assembly 2012** recommends the use of `au`
* **[The International System of Units. supplement 2014](https://www.bipm.org/utils/common/pdf/si_supplement_2014.pdf). In: bipm.org. 2014** recommends the use of `au`, referring to the IAU resolution
* [en Wikipedia](https://en.wikipedia.org/wiki/Astronomical_unit#History_of_symbol_usage) (in addition to `au` and `AU`) also lists `ua` referring to ISO 80000-3:2006, which got replaced by ISO 80000-3:2019 not stating this anymore
* [de Wikipedia](https://de.wikipedia.org/wiki/Astronomische_Einheit) (in addition to `au`, `AU` and `ua`) further states `AE` as a symbol used in German language literature, but without reference

**Comment by HajoRijgersberg** [comment]
Just defined 'au' as the symbol of the astronomical unit and added 'AU' as an alternative symbol!
Thanks so much, Jan Martin, for diving into this! :)

**Comment by JeanEudesH** [comment]
I reuse this issue to point the case of the pixel unit. There is no symbol declared but two can be used "px" and "p". The solution adopted here seems to fit well in this other case.
http://www.ontology-of-units-of-measure.org/resource/om-2/pixel

**Comment by dr-shorthair** [comment]
Pixel is not a unit-of-measurement. 'pixel-count' might be, but of course the quantity here is 'count' or 'number', with the annotation 'of pixels'.

**Comment by JeanEudesH** [comment]
To me the pixel is a unit used to measure the dimension of images and the definition of a screen. 
The exact value of a pixel in number of cm2 or mm2 is not constant so I understand It can be tricky to define pixel as a unit.

The pixel is also registred as "om:unit" and "om:singularUnit". Is this intended ? I haven't fully understood the classes of this ontology yet, but this looks like "base" units before any prefix, squared or any combinaison.

**Comment by dr-shorthair** [comment]
If the conversion to a standard unit (m) for the same quantity-kind (length) is variable, then it does not belong in a list of standardized units-of-measurement.

**Comment by jmkeil** [comment]
Unfortunately, the situation is not that easy. E.g. starting from [CSS level 2](https://www.w3.org/TR/CSS2/syndata.html#length-units) (and also in [level 3](https://drafts.csswg.org/css-values-3/#absolute-lengths) and [level 4 draft](https://drafts.csswg.org/css-values-4/#absolute-lengths)), in CSS *pixel* is a absolute unit of length with defined conversion of `1px = 1/96th of 1in`. However, this distinguishes from the (IMO) common perception of *pixel* as a unit for the resolution of images, screens, cameras, …. For this notion of *pixel* I agree to @dr-shorthair that it actually is a count. Especially, as the product of two pixel values is still called a pixel value, not a pixel square value. E.g: A resolution is typically represented this way: `1920 px × 1080 px = 2.073.600 px (not px²) ≈ 2.07 MP`.

**Comment by HajoRijgersberg** [comment]
Hi guys,
Sorry for my late reaction, very busy times at my work...
Thanks for your effort and discussion! :)
Hope to clear anything: Seems like pixel is indeed a unit, defined as 1/96th of an inch. The unit is included in OM, but indeed I have not yet specified the symbol (it seems that this should be px, if I look at the literature sources). I also haven't defined the conversion yet. I am certainly willing to do that. Would you be helped with that, Jean?
Pixel count is indeed a quantity, also present in OM.
Looking forward to your reply!
Cheers, Hajo

**Comment by dr-shorthair** [comment]
... except (as [@jmkeil notes](https://github.com/HajoRijgersberg/OM/issues/40#issuecomment-792846841)) there is ambiguity about whether it is area or length. For me, pixel is too overloaded to be a real unit-of-measurement, but I guess I just won't use it then ;-)

**Comment by JeanEudesH** [comment]
That was my initial thought when starting, but after discussion and @jmkeil reply it is not that easy indeed! 
The most common use of this unit will be the pixel count for images and screen (at least that's what I intended to use it for). 
I will use some local definition and link it to yours instead

**Comment by HajoRijgersberg** [comment]
Indeed, it's strange that it is not clear whether this unit is a unit of length or area. Thanks for pointing this out!
So, I'll leave pixel as it is in OM. In OM namely it is not decided whether it is a length or area unit. It is simply not indicated there; it is an independent unit, only used for the quantity 'pixel count'. As that quantity already indicates, it is not a length or an area; it is a count.
So, Jean, if you want, I think you can use OM's pixel and pixel count for your purpose.
I think you're correct that the main use of pixel is for counting pixels in images and screens.
Hope this helps and looking forward to everyone's reply again! And thanks for all your effort! :)
